docs/hazmat/backends/openssl.rst

```diff
@@ -68,6 +68,11 @@ greater.
 OS random engine
 ----------------
 
+.. note::
+
+    As of OpenSSL 1.1.1d the CSPRNG is fork-safe by default. ``cryptography``
```

Comment:
This should clarify that it's _OpenSSL's builtin CSPRNG_ which is now fork-safe.